### ISSUE TITLE
RFC checklist

### ISSUE THREAD

**Issue by Ovid** [Ovid/Cor#28]
This is undoubtedly incomplete, but the following items will be needed to be accomplished before we can get an RFC to P5P:

- [x] ~~`:handles(@%*)`. The `:handles(*)` syntax allows all unknown method to be delegated to this slot (we can fake inheritance from non-Corinna objects this way)~~
- [x] Do we [rename `has`](https://github.com/Ovid/Cor/issues/4?fbclid=IwAR2-g3xLyj_JPoQxfjcaaEPq8Nft1WSaGO7r8NYGOVGIRY14DYgVEvELJTo#issuecomment-873387759)?
- [x] Declaration of [class slots and methods](https://github.com/Ovid/Cor/issues/35).
- [ ] Document Moo/se idioms in Corinna (maybe toss in stuff from other popular OO systems?)
- [ ] [Disambiguating Corinna objects from other objects](https://github.com/Ovid/Cor/issues/32)
- [x] [Twigils or something else](https://github.com/Ovid/Cor/issues/29)?
- [x] Get signoff for [Overview](https://github.com/Ovid/Cor/blob/master/rfc/overview.md)
- [x] Get signoff for [Grammar](https://github.com/Ovid/Cor/blob/master/rfc/grammar.md)
- [x] Get signoff for [Classes](https://github.com/Ovid/Cor/blob/master/rfc/classes.md)
- [x] Get signoff for [Class Construction](https://github.com/Ovid/Cor/blob/master/rfc/class-construction.md)
- [x] Get signoff for [Attributes](https://github.com/Ovid/Cor/blob/master/rfc/attributes.md)
- [x] Get signoff for [Methods](https://github.com/Ovid/Cor/blob/master/rfc/methods.md)
- [x] Get signoff for [Roles](https://github.com/Ovid/Cor/blob/master/rfc/roles.md)
- [x] Get signoff for [Phasers](https://github.com/Ovid/Cor/blob/master/rfc/phasers.md)
- [x] Get signoff for [Method Modifiers](https://github.com/Ovid/Cor/blob/master/rfc/method-modifiers.md)

**Comment by Ovid** [comment]
@leonerd I've assigned this to you not because I expect you to do all of this, but because I wanted to be sure that you're away of this checklist and can mark off anything as completed if that time arises.

**Comment by Ovid** [comment]
For now, skipping documenting differences between Corinna and `Object::Pad` because the latter is a fast moving target and the documentation will always be out of date.

**Comment by Ovid** [comment]
Also skipping the following two because these are important, but they're part of the implementation, not part of the RFC:

* State inspection tools. It should be human-readable, but not something you should rely on programmatically. This relates to [the debugger and Data::Dumper](https://github.com/Ovid/Cor/issues/30) (I believe @leonerd has already done much of this)
* Ensure the full [Corinna construction phases](https://gist.github.com/Ovid/accb0c7c8444bdd150b5c7509809477f) are implemented

**Comment by sigzero** [comment]
This might be dumb. I was thinking about your "module" proposal. Since we already have "package" why not use "parcel" as the keyword. It's not used anywhere. Parcel reminds me of a smaller package. Maybe not the best place to share this but I wasn't sure where else.

**Comment by Ovid** [comment]
@sigzero Just saw this, sorry.

`parcel`'s an interesting name. However, for now, the PSC has passed on the module proposal. If there's something more compelling in the future, we can revisit this.
